app/controllers/cms_controller.rb

```diff
@@ -36,11 +22,51 @@ def clear_page_cache
     redirect_to request.fullpath.sub(%r{/refresh$}, "")
   end
 
+  def privacy
+    process_resource Cms::Pages::PrivacyNotice
+  end
+
+  def deep_test
+    process_resource Cms::Pages::DeepTest
+  end
+
   private
 
+  def process_collection(cls, title: "News & Updates", page_name: "Articles", collection_wrapper: "ncce-news-archive")
```

Comment:
Should process_collection and process_resource live in a concern so we can use them in other controllers?

Reply:
Good idea!